smdebug/tensorflow/keras.py

```diff
@@ -139,9 +140,16 @@ def _is_not_supported(self):
                 self._hook_supported = False
             elif self.distribution_strategy == TFDistributionStrategy.MIRRORED:
                 try:
-                    from tensorflow.python.keras.distribute.distributed_training_utils import (
-                        get_distributed_model,
-                    )
+                    if is_tf_version_greater_than_2_4_x():
```

Comment:
Can we replace [this function from utils](https://github.com/awslabs/sagemaker-debugger/blob/master/smdebug/tensorflow/utils.py#L24) with this function defined here to reduce code redundancy?